2022/09/13 09:52:09 [emerg] 16120#18744: CreateFile() "E:\2022新版课程\苍穹外卖项目\授课资料\day01\资料\前端运行环境\nginx-1.20.2/conf/nginx.conf" failed (1113: No mapping for the Unicode character exists in the target multi-byte code page)
2022/09/19 14:13:10 [emerg] 6884#1840: CreateFile() "E:\2022新版课程\苍穹外卖项目\授课资料\day01\资料\前端运行环境\nginx-1.20.2/conf/nginx.conf" failed (1113: No mapping for the Unicode character exists in the target multi-byte code page)
2023/03/04 10:52:01 [emerg] 12092#12960: CreateFile() "E:\授课\北京JavaEE就业406期\苍穹外卖项目\资料\day01\前端运行环境\nginx-1.20.2/conf/nginx.conf" failed (1113: No mapping for the Unicode character exists in the target multi-byte code page)
2025/04/08 17:43:59 [error] 26772#21432: *2 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0706348f_ac99_4c13_8fa1_4740a9756da7__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:43:59 [error] 26772#21432: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0fb5fbd7_6d31_4084_b5bb_2e9323056e97__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:45:48 [error] 26772#21432: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:46:40 [error] 26772#21432: *11 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0578a1cb_9982_4580_baf9_f45097e3b15e__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:46:40 [error] 26772#21432: *11 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__84570c87_1847_46de_8c2b_d00ff098e5c9__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:47:02 [error] 26772#21432: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:47:02 [error] 26772#21432: *1 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:47:09 [error] 26772#21432: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0b77af7e_c4fd_4959_a1ca_0e0f3d1eaade__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 17:47:09 [error] 26772#21432: *11 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__924496e4_e719_4c06_a95e_0a68b0640c0f__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:16:50 [error] 26772#21432: *35 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__30e867cc_ef1c_40ab_a7a1_1ea5d5eadccf__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:16:50 [error] 26772#21432: *31 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__9716cb6e_15c5_4f96_9411_8cce3b89ed43__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:17:57 [error] 26772#21432: *31 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__537aa338_88a8_4320_b2df_e4fde6b4311f__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:17:57 [error] 26772#21432: *31 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__b99d72b7_11b6_4461_ae41_3852e9c83a1a__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:18:00 [error] 26772#21432: *31 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__cc6d6a30_4b04_4796_9bda_6005b2895f51__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:18:00 [error] 26772#21432: *46 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d5502921_9beb_41fb_8baa_5cc73a30681e__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:19:21 [error] 26772#21432: *48 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__45f7823e_3f87_451f_8303_36e5b868f685__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:19:21 [error] 26772#21432: *49 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__a1747241_1a5e_4c90_8bdd_9427481e71bf__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:20:22 [error] 26772#21432: *49 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:07 [error] 26772#21432: *52 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f9e8f113_f1e6_4bcb_b1f3_a01d7a8d5f3a__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:07 [error] 26772#21432: *53 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__aa4e063c_8272_47d0_95df_dcdcfbe4ad70__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:13 [error] 26772#21432: *53 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4fbe9261_7a46_4e35_abb3_026026cd841c__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:13 [error] 26772#21432: *52 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__22c8d0df_6125_4059_8c5f_74cac461b442__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:20 [error] 26772#21432: *52 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0fbdcf01_3162_4478_8bc9_80c3a4618629__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:22:20 [error] 26772#21432: *53 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d7a0c5b9_3b40_42e2_a02c_0d308b342363__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:23:21 [error] 26772#21432: *53 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:09 [error] 26772#21432: *60 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:11 [error] 26772#21432: *60 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:18 [error] 26772#21432: *60 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://localhost/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:19 [error] 26772#21432: *60 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/employee/login HTTP/1.1", upstream: "http://localhost/admin/employee/login", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:20 [error] 26772#21432: *60 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__efa9c60d_79c5_4e4b_9e22_1df6681514ab__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 19:29:20 [error] 26772#21432: *60 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4574c355_e3f4_408d_809a_235044f50b57__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 20:17:54 [error] 26772#21432: *99 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__58f432d5_c6a3_4aab_9857_b8035a11fcc5__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 20:17:54 [error] 26772#21432: *101 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4620dc2d_3077_4295_9d3c_407db8aedbc5__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 21:04:03 [error] 26772#21432: *114 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7c6ae022_4b0a_4b7d_b57e_2fc17fedebc7__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/08 21:04:03 [error] 26772#21432: *115 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7663e49b_efed_4470_a04e_4e6c4c58da10__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:28:31 [error] 17600#16332: *2 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__86b2a61d_dafc_4409_a197_6cabecad2aca__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:28:31 [error] 17600#16332: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c54d2ca9_283e_4033_8a9c_ddda3174abaf__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:29:54 [error] 17600#16332: *22 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__fd259bee_797f_4070_9874_593991b33d5e__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:29:54 [error] 17600#16332: *22 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0d8eb91a_b592_4bcb_a905_0dbe1d4cc1dc__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:31:33 [error] 17600#16332: *38 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__95aac66d_370b_45c2_9bf2_a120c020886c__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:31:33 [error] 17600#16332: *38 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__03ad90c3_8c1b_446d_9394_19deb6383558__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:31:43 [error] 17600#16332: *34 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__17717e49_7eb3_436a_9872_487c9827f7c1__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:31:43 [error] 17600#16332: *34 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__54064faa_1fa3_4d82_884d_4f0d08eb1024__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:32:55 [error] 17600#16332: *64 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__2e8137b1_f385_48a2_a721_35032a07d4b9__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:32:55 [error] 17600#16332: *64 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__707e3670_51c1_4b1a_918b_7196e287dd16__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:34:24 [error] 17600#16332: *87 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__a688950c_7b77_4b16_aeb0_20d2ef4ebc36__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:34:24 [error] 17600#16332: *87 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__49a7e6a3_3ea2_4f08_aa12_7ddfcabb70e7__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:39:36 [error] 17600#16332: *105 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0297d0e9_aadc_4eaf_853a_e10d601fe195__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:39:36 [error] 17600#16332: *105 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__ec572602_d612_492f_adea_87b1e28910c3__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:41:39 [error] 17600#16332: *119 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7210a803_0bdf_4ae2_bb5a_2c075d93d4f1__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:41:39 [error] 17600#16332: *120 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__8e9dc853_9828_4d74_9bdb_85b19e684c14__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:13 [error] 17600#16332: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__af50d50d_d0be_4531_9417_7bfd3116c7b3__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:13 [error] 17600#16332: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__8736d41d_d656_4ee6_91ae_3fddf7787e51__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:35 [error] 17600#16332: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1de6ca81_432a_4e4c_aa22_c869567d8454__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:35 [error] 17600#16332: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__36ca2bbf_0eb3_4feb_8365_1f1e8dfb5f51__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:42 [error] 17600#16332: *128 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__e7d821be_cbd9_4c8a_93bf_c39bf0c12697__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:44:42 [error] 17600#16332: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__04a1866f_7707_42b6_8a0b_68345e9ebc8d__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:48:41 [error] 17600#16332: *143 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__75524be7_78f8_40c3_951b_5ff10ef7043c__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 17:48:41 [error] 17600#16332: *142 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__e0dd7cca_b079_4688_8287_a8173bc05024__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:15 [error] 17600#16332: *154 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__12e84334_a852_4704_826f_7ebad7c72606__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:15 [error] 17600#16332: *154 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__8c782f47_d85f_4ff8_b452_e19b1ed39721__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:17 [error] 17600#16332: *151 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:17 [error] 17600#16332: *150 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:17 [error] 17600#16332: *150 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://localhost/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:17 [error] 17600#16332: *155 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/y6paynk5q3e HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/y6paynk5q3e", host: "localhost"
2025/04/09 18:59:19 [error] 17600#16332: *151 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://[::1]:8080/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:25 [error] 17600#16332: *151 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://localhost/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:25 [error] 17600#16332: *150 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:25 [error] 17600#16332: *150 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7d2def42_9bf3_442a_ba77_dc6ebe59fa99__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:25 [error] 17600#16332: *151 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f5ac48dc_6f36_40a0_94e4_c478b5805515__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:28 [error] 17600#16332: *150 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:28 [error] 17600#16332: *150 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__3a6a2a66_e050_4312_ba91_e95c9099008a__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 18:59:28 [error] 17600#16332: *154 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d833b645_fbdb_4364_bb13_ecff04b6cf5c__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:32:35 [error] 17600#16332: *170 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f24464c2_8a79_4990_8a48_a820ee80144e__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:32:35 [error] 17600#16332: *170 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__07257bb4_ab55_476a_8481_7e188abfb4eb__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:09 [error] 17600#16332: *181 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__561fe256_2df7_487d_bbd4_b7689ae57276__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:09 [error] 17600#16332: *181 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__fa66c7bd_da95_4537_8b28_a235d1cf59b1__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:11 [error] 17600#16332: *177 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:11 [error] 17600#16332: *178 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:11 [error] 17600#16332: *178 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/09 19:35:12 [error] 17600#16332: *182 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/82xc3ddbijq HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/82xc3ddbijq", host: "localhost"
2025/04/09 19:35:13 [error] 17600#16332: *177 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/09 20:11:56 [error] 17600#16332: *188 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1241b5d6_bdf5_4fc2_9510_be5d2563bcef__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 20:11:56 [error] 17600#16332: *188 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d75e2166_6578_490d_97a3_4e34b433176c__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 20:31:14 [error] 17600#16332: *215 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__133c27a4_e42f_4de9_96a8_22e42c9a20b4__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 20:31:14 [error] 17600#16332: *215 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c918847c_c904_47b5_a9a4_1bbd3f14620c__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/09 21:44:22 [alert] 10348#22652: the event "ngx_master_10348" was not signaled for 5s
2025/04/09 21:44:22 [alert] 10348#22652: could not respawn worker
2025/04/10 20:44:47 [error] 4056#23964: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__b23be4e9_1307_4f16_86b3_f5e70c1221dd__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 20:44:47 [error] 4056#23964: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d45f2368_9095_477b_8130_57d3d02eccb8__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 20:46:03 [error] 4056#23964: *2 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:12:53 [error] 4056#23964: *20 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__d4f8efbb_6891_42fa_93cc_e4447566bc5c__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:12:53 [error] 4056#23964: *20 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f6859532_dd97_4917_bff1_ce25081bb536__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:09 [error] 4056#23964: *20 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__5e274bc9_4ed8_46a9_81e4_d9d47c281fb7__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:09 [error] 4056#23964: *20 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__43de97d5_d00d_49e4_b3ac_6758ff50f0c5__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:30 [error] 4056#23964: *48 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:31 [error] 4056#23964: *51 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:31 [error] 4056#23964: *51 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://localhost/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:13:32 [error] 4056#23964: *48 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://[::1]:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:16:09 [error] 4056#23964: *58 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f9cd21fe_297e_4ef2_9079_00a443ffc50b__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:16:09 [error] 4056#23964: *58 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4f96cc00_91cb_4fe3_8317_2e771723a89c__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:18:28 [error] 4056#23964: *64 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__6042ca30_154a_4fc3_b35f_9281b9f29ebd__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:18:28 [error] 4056#23964: *64 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4a7172ad_adc3_4a60_a0db_512bf68e37c2__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:20:11 [error] 4056#23964: *78 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__9d0eb7b1_e046_4470_b2f8_4bc70a7a34d0__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:20:11 [error] 4056#23964: *78 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__32af53f0_2886_41fe_855a_cd1775fd346e__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:21:25 [error] 4056#23964: *76 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:22:25 [error] 4056#23964: *76 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://[::1]:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:26:54 [error] 4056#23964: *88 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:27:58 [error] 4056#23964: *88 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:28:58 [error] 4056#23964: *88 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "PUT /api/employee HTTP/1.1", upstream: "http://[::1]:8080/admin/employee", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:17 [error] 4056#23964: *95 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://[::1]:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:18 [error] 4056#23964: *105 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c8994e65_4812_4acf_9e63_671caf8cc963__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:18 [error] 4056#23964: *105 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c218e164_c80c_4c6f_8999_a00d00e804c9__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:20 [error] 4056#23964: *101 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:20 [error] 4056#23964: *101 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/employee/page?page=1&pageSize=10 HTTP/1.1", upstream: "http://localhost/admin/employee/page?page=1&pageSize=10", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:20 [error] 4056#23964: *103 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/w6t1x1rwrac HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/w6t1x1rwrac", host: "localhost"
2025/04/10 21:31:20 [error] 4056#23964: *105 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/10 21:31:20 [error] 4056#23964: *105 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://localhost/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/11 11:12:10 [error] 3856#16020: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__874f31fe_f8fb_4f0b_81e6_a15df4c86140__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 11:12:10 [error] 3856#16020: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7f02a20a_392f_44ac_b34b_fcc790a2d1ed__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 11:13:57 [error] 3856#16020: *2 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/common/upload HTTP/1.1", upstream: "http://[::1]:8080/admin/common/upload", host: "localhost", referrer: "http://localhost/"
2025/04/11 11:21:43 [error] 3856#16020: *21 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__8e3fbe23_3122_433c_867f_1efbc577caec__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 11:21:43 [error] 3856#16020: *20 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__72ee968b_8b73_4e1a_aa4e_7d33ba1a0728__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:17:20 [error] 3856#16020: *29 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1982eb8d_68b2_445b_b30e_76ddd442c8fb__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:17:20 [error] 3856#16020: *29 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1ce98e2f_1d0e_42f4_94ec_c3e24c37f6ca__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:35:39 [error] 3856#16020: *58 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:41:53 [error] 3856#16020: *61 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://[::1]:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:43:12 [error] 3856#16020: *64 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:43:14 [error] 3856#16020: *64 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://[::1]:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:45:23 [error] 3856#16020: *68 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://[::1]:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:46:48 [error] 3856#16020: *68 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:54:17 [error] 3856#16020: *72 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://[::1]:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 12:54:54 [error] 3856#16020: *72 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/dish HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish", host: "localhost", referrer: "http://localhost/"
2025/04/11 16:38:13 [error] 3856#16020: *98 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__f8108a17_f726_4b43_88ae_3d6b14bff577__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 16:38:13 [error] 3856#16020: *98 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__fa00dad0_18ad_42b0_9db9_2645ee5e55b8__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 16:42:18 [error] 3856#16020: *127 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4bb0a817_7d2f_4561_877a_b6224851d590__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 16:42:18 [error] 3856#16020: *128 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__46727a19_d087_46ae_b200_2d9d52387d81__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 17:37:18 [error] 3856#16020: *140 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__69e55bcb_ff5c_44f7_a44c_dde72eff0721__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 17:37:18 [error] 3856#16020: *140 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__eb20450a_9972_43b5_bbb6_9593358b7199__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 17:56:34 [error] 3856#16020: *171 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "DELETE /api/dish?ids=75,74 HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish?ids=75,74", host: "localhost", referrer: "http://localhost/"
2025/04/11 17:57:34 [error] 3856#16020: *171 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "DELETE /api/dish?ids=75,74 HTTP/1.1", upstream: "http://[::1]:8080/admin/dish?ids=75,74", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:09:36 [error] 3856#16020: *181 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "DELETE /api/dish?ids=73,75 HTTP/1.1", upstream: "http://[::1]:8080/admin/dish?ids=73,75", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:12:47 [error] 3856#16020: *188 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "DELETE /api/dish?ids=73,75 HTTP/1.1", upstream: "http://[::1]:8080/admin/dish?ids=73,75", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:47 [error] 3856#16020: *208 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c6261b77_775d_49c3_a19e_7d620c1c1c52__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:47 [error] 3856#16020: *209 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__05d960c1_eb6a_44a9_89a3_86d3c132a98e__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:49 [error] 3856#16020: *202 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/shop/status HTTP/1.1", upstream: "http://[::1]:8080/admin/shop/status", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:49 [error] 3856#16020: *200 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/dish/page?page=1&pageSize=10&status= HTTP/1.1", upstream: "http://[::1]:8080/admin/dish/page?page=1&pageSize=10&status=", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:49 [error] 3856#16020: *201 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/category/list?type=1 HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/category/list?type=1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:49 [error] 3856#16020: *201 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/category/list?type=1 HTTP/1.1", upstream: "http://localhost/admin/category/list?type=1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:49 [error] 3856#16020: *206 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ws/0as67w3xdksh HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/0as67w3xdksh", host: "localhost"
2025/04/11 18:30:51 [error] 3856#16020: *200 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/dish/page?page=1&pageSize=10&status= HTTP/1.1", upstream: "http://localhost/admin/dish/page?page=1&pageSize=10&status=", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:30:51 [error] 3856#16020: *202 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/category/list?type=1 HTTP/1.1", upstream: "http://localhost/admin/category/list?type=1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:34:01 [error] 3856#16020: *238 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__69735d16_d930_4384_a2f0_8fd0e9b02985__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:34:01 [error] 3856#16020: *239 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__aed68fb1_8cf4_4d83_aa94_0a0956c7ecfd__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:34:03 [error] 3856#16020: *247 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__9ca441d7_bd54_4a39_9b43_10337e646141__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:34:03 [error] 3856#16020: *247 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__bf5506b6_4c77_461f_b225_ff0c0676c267__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:54:01 [error] 3856#16020: *264 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/category/list?type=1 HTTP/1.1", upstream: "http://[::1]:8080/admin/category/list?type=1", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:54:01 [error] 3856#16020: *263 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/dish/page?page=1&pageSize=10&status= HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/dish/page?page=1&pageSize=10&status=", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:54:01 [error] 3856#16020: *263 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/dish/page?page=1&pageSize=10&status= HTTP/1.1", upstream: "http://localhost/admin/dish/page?page=1&pageSize=10&status=", host: "localhost", referrer: "http://localhost/"
2025/04/11 18:54:03 [error] 3856#16020: *264 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/category/list?type=1 HTTP/1.1", upstream: "http://127.0.0.1:8080/admin/category/list?type=1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:40:03 [error] 12888#3636: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__04d9a724_736d_4055_88a5_f90ba2f77247__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:40:03 [error] 12888#3636: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__83fc7628_4ad6_473f_add5_8267e36e3273__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:40:22 [error] 12888#3636: *14 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7f8fdd1e_c083_446d_99c9_a8a36d7e28c0__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:40:22 [error] 12888#3636: *14 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c066298f_a1af_4732_ab62_8a1ba87f9a6d__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:41:23 [error] 12888#3636: *12 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__16b14ead_8872_4201_88a8_af02a0c316ea__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 13:41:23 [error] 12888#3636: *12 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__3c4a6161_7ab9_4e7e_a15b_3878fc6a07b2__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:02:43 [error] 12888#3636: *43 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__05bcb1a8_53ca_48da_a478_91b453a3d2de__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:02:43 [error] 12888#3636: *43 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__78d03063_fce9_469f_b944_1fcd4afd11b1__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:02:58 [error] 12888#3636: *44 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__a3cfe41b_1082_4361_aaca_c755c67d4e13__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:02:58 [error] 12888#3636: *44 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__a102abb9_6982_4fba_a2aa_a3d3d2741897__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:04:41 [error] 12888#3636: *48 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1e5fe495_d4bc_4b13_bee4_e5977185be72__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 17:04:41 [error] 12888#3636: *48 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__eef4e6b6_e195_4893_9680_3386e658bf22__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:02:22 [error] 12888#3636: *67 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__8446905b_839a_4b05_84c2_56d3e15164f8__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:02:22 [error] 12888#3636: *67 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1ecd8e2d_df38_40b2_ba48_0919150fa44c__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:11:48 [error] 12888#3636: *116 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__15cf86b6_ff38_43d9_8442_62567e92cf26__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:11:48 [error] 12888#3636: *116 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__92286c4b_65bc_42e8_864d_4fb70713168f__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:21:02 [error] 12888#3636: *175 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__5c7915ed_bd1a_4d95_81b0_699911e42452__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:21:02 [error] 12888#3636: *175 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__ca59026c_88d3_4be5_8767_34ee15d41852__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:21:42 [error] 12888#3636: *175 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__caa3c679_9cf2_468f_b803_6cadff16fd04__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:21:42 [error] 12888#3636: *175 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7e4f5d10_de81_4c43_aee4_7d8e55290ac6__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:07 [error] 12888#3636: *220 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__7e88c98d_cf98_4d16_9613_49afc47843f8__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:07 [error] 12888#3636: *221 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__6c705e1f_9a6e_4297_a0a7_389c24eea3a6__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:12 [error] 12888#3636: *216 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1d8a91aa_8285_45ee_b305_27549e1e511b__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:12 [error] 12888#3636: *216 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__95127654_bded_43e1_b08d_5ad2aadee665__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:24 [error] 12888#3636: *213 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__fd362d0a_ed00_4861_8f05_6a07b82c6ff2__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 18:34:24 [error] 12888#3636: *213 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0d8a6afd_3e03_43fd_9c55_1489ddd72461__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 19:29:20 [error] 12888#3636: *280 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__38126f1b_1f15_4036_80d1_7ef316ef97af__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/13 19:29:20 [error] 12888#3636: *280 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__27694936_e149_4ff7_a18d_71c1f25d039a__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:02:00 [error] 3792#4676: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__e4e48d37_6ba7_491c_af73_4da3a96af80a__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:02:00 [error] 3792#4676: *2 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__03936ad1_ed54_4d37_a55c_5cc040e0bf83__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:05:51 [error] 3792#4676: *1 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__cbaea75c_d37c_43ba_87eb_f5ab1829a5d8__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:05:51 [error] 3792#4676: *2 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__c2a99c08_cf52_4a45_9e6d_2588488587da__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:22:48 [error] 3792#4676: *203 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__dfd6e95a_489c_4d4d_bfd6_c35669ae72ca__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:22:48 [error] 3792#4676: *203 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__859b42c9_1ad8_4429_8507_8a2435f176ec__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:31:39 [error] 3792#4676: *211 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__cb7c19b6_6659_44a4_b5dc_3f353f3cdc55__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 16:31:39 [error] 3792#4676: *211 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__0cc08a27_822c_4f87_adbd_dd996ce72769__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 17:19:25 [error] 3792#4676: *263 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__1dcba733_4f99_43b5_a2c1_79f48b899848__0 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2025/04/14 17:19:25 [error] 3792#4676: *263 CreateFile() "E:\zjut_Food_Delivery_Ordering_Program\frontend\nginx-1.20.2/html/sky/hybridaction/zybTrackerStatisticsAction" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /hybridaction/zybTrackerStatisticsAction?data=%7B%7D&__callback__=__cb__zybTrackerStatisticsAction__4e951030_9065_44b5_9844_c9055eca7a4e__1 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
